Research/ResearchUI/iOS/Step View Controllers/RSDScrollingOverviewStepViewController.swift

```diff
@@ -62,10 +75,17 @@ open class RSDScrollingOverviewStepViewController: RSDOverviewStepViewController
     @IBOutlet
     open var infoButton: UIButton!
     
+    /// The button that when pressed displays an instuctional video.
+    @IBOutlet
+    open var seeThisInActionButton: UIButton!
```

Comment:
I don't see where you are hiding this button if there isn't a video? Also, where are you setting the localizable text for this button?

Reply:
Both of those need done, I'll implement as part of the RSDVideoUIAction refactor.